docs/source/content/supervised/logistic_regression.rst

```diff
@@ -5,52 +5,136 @@ Logistic Regression
 Introduction
```

Comment:
Can you add a table of contents at the top of this like the overfitting section?

Reply:
9185ec3